android/sdk/src/main/java/com/taobao/weex/WXSDKInstance.java

```diff
@@ -716,7 +719,7 @@ private void renderInternal(String pageName,
 
     final IWXJscProcessManager wxJscProcessManager = WXSDKManager.getInstance().getWXJscProcessManager();
 
-    if(wxJscProcessManager != null && wxJscProcessManager.shouldReboot()) {
+    if(wxJscProcessManager != null && wxJscProcessManager.shouldReboot() && !skipFrameworkInit()) {
```

Comment:
I don't think you need changing this line.

The `if` case you changing is about a jscReboot mechanism, which will reboot JSC if **Weex Render** doesn't finished in the given `rebootTimeout()`.  The reboot mechanism mainly used to prevent deadlock of JSC, which has no relationship with dataRender.